lib/bolt_ext/schemas/task.json

```diff
@@ -31,9 +31,13 @@
               "description": "The type the parameter should accept"
             }
           }
+        },
+        "input_method": {
+          "type": "string",
+          "enum": ["stdin", "environment"],
```

Comment:
There are other options. For Windows, `powershell` is valid.

Reply:
https://github.com/puppetlabs/puppet-specifications/tree/master/tasks#input-methods

Reply:
OK, so maybe we just leave out `input_method` and remove the additionalProperties bit? Looks like you did that as part of #651. @adreyer Asked earlier if we should be validating anything in metadata? For example if the task description is an integer would that really matter?